index.d.ts

```diff
@@ -0,0 +1,173 @@
+/**
+ * This should be removed once added upstream:
+ * https://github.com/TylerBarnes/gatsby-plugin-transition-link/pull/202
+ */
+
+declare module 'gatsby-plugin-transition-link/AniLink' {
+	import { Component } from 'react';
+
+	interface AniLinkFade {
+		fade: boolean;
+	}
+	type AniLinkPaintDripColors =
+		| {
+				color?: string;
+		  }
+		| {
+				hex?: string;
+		  };
+	type AniLinkPaintDrip = AniLinkPaintDripColors & {
+		paintDrip: boolean;
+	};
+	interface AniLinkSwipe {
+		swipe: boolean;
+	}
+	interface AniLinkCover {
+		cover: boolean;
+		bg?: string;
+	}
+	type AniLinkTypes = AniLinkFade | AniLinkPaintDrip | AniLinkSwipe | AniLinkCover;
+
+	interface AniLinkProps {
+		to: string;
+		direction?: string;
+		duration?: number;
+		top?: 'exit' | 'entry';
+		entryOffset?: number;
+		title?: string;
+		target?: string;
+	}
+	class AniLink extends Component<AniLinkProps & AniLinkTypes> {}
```

Comment:
```txt
Type '{ children: ReactNode; fade: true; to: string; }' is not assignable to type 'IntrinsicAttributes & (IntrinsicClassAttributes<AniLink> & Readonly<AniLinkProps & AniLinkTypes>)'.
      Property 'children' does not exist on type 'IntrinsicAttributes & (IntrinsicClassAttributes<AniLink> & Readonly<AniLinkProps & AniLinkTypes>)'.
```

Solved by adding children to the declaration:

```suggestion
	class AniLink extends Component<
  React.PropsWithChildren<AniLinkProps> & AniLinkTypes
  > {}
```